sympy/utilities/tests/test_lambdify.py

```diff
@@ -771,3 +772,10 @@ def test_Indexed():
     i, j = symbols('i j')
     b = numpy.array([[1, 2], [3, 4]])
     assert lambdify(a, Sum(a[x, y], (x, 0, 1), (y, 0, 1)))(b) == 10
+
+def test_issue_12173():
+    #test for issue 12173
+    exp1=lambdify((x,y), uppergamma(x,y),"mpmath")(1,2)
```

Comment:
SymPy code formatting expects that `=` (and `==`) have a space on either side. There should also be a space after each comma.